tests/unit/s2n_tls13_cert_verify_test.c

```diff
@@ -302,7 +303,6 @@ int run_tests(const struct s2n_tls13_cert_verify_test *test_case, s2n_mode verif
         /* send and receive with mismatched signature algs */
         verifying_conn->handshake_params.client_cert_sig_scheme = &test_scheme;
```

Comment:
Can you create a second test_scheme for this second test case? I think one of the reasons why this test is so difficult to figure out is that we're reusing this test scheme struct for both cases and modifying it both times, which is super weird.

Also along those lines, since the first test case changes verifying_conn->handshake_params.client_cert_sig_scheme, can you set it back to what it originally was before starting the first test case? Maybe that will make it more clear what is going on here.